nautobot/core/choices.py

```diff
@@ -182,14 +182,14 @@ class ButtonActionColorChoices(ChoiceSet):
     Map standard button actions to Bootstrap color classes.
     """
 
-    ADD = "info"
+    ADD = "primary"
```

Comment:
Needed to make sidebar "add" buttons not turn gray. Also now consistent with `IMPORT = "primary"` below.